Kickstarter-iOS/Features/PledgeView/Views/EstimatedShippingCheckoutView.swift

```diff
@@ -0,0 +1,42 @@
+import Library
+import SwiftUI
+
+struct EstimatedShippingCheckoutView: View {
+  let estimatedCost: String
+  let aboutConversion: String
+
+  private enum Constants {
+    public static let maxHeight = 125.0
+    public static let defaultSpacing = Styles.grid(1)
+    public static let verticalPadding = Styles.grid(2)
+    public static let horizontalPadding = Styles.grid(4)
+  }
+
+  // TODO: Update strings with translations [mbl-1667](https://kickstarter.atlassian.net/browse/MBL-1667)
+  var body: some View {
+    HStack(alignment: .top) {
+      VStack(alignment: .leading, spacing: Constants.defaultSpacing) {
+        Text("Estimated Shipping")
+          .font(Font(UIFont.ksr_subhead(size: 16)))
```

Comment:
I think it would be helpful to define typography and define enum sizes to avoid hardcoded numbers.